app/scripts/dimApp.i18n.js

```diff
@@ -203,6 +203,9 @@
             Discipline: "Discipline",
             Intellect: "Intellect",
             NoBonus: "No Bonus",
+            OfMaxRoll: "of max roll",
```

Comment:
This seems like it should be a full sentence with a substitution rather than a fragment (something like `{{range}} of max roll`